data/en/1_0005.json

```diff
@@ -44,136 +44,144 @@
+                                "benefits employees receive through pay, for example, family working tax credit",
+                                "expenses payments for attending meetings, for example, councillors"
+                            ]
+                        }]
+                    }, {
+                        "question": "Holiday pay, pay awards and bonuses",
+                        "content": [{
+                            "description": "Include:",
+                            "list": [
+                                "performance pay (for example productivity bonuses)",
+                                "long service awards",
+                                "appearance money (sporting professionals)"
+                            ]
                         }, {
-                            "question": "Redundancies and temporary employees",
-                            "content": [{
-                                "description": "Whether there were any significant changes to the:",
-                                "list": [
-                                    "number of redundancies among paid employees",
-                                    "number of temporary paid employees",
-                                    "total gross pay paid to employees"
-                                ]
-                            }, {
-                                "description": "Include:",
-                                "list": [
-                                    "more or less overtime",
-                                    "new pay rates",
-                                    "industrial action"
-                                ]
-                            }]
-
+                            "description": "Exclude:",
+                            "list": [
+                                "trainees on government schemes",
+                                "director’s fees",
+                                "employer’s NI and contribution to pension schemes",
+                                "employees working abroad unless paid directly from this business’s GB payroll",
+                                "signing on fees (sporting professionals)",
+                                "payment in lieu of notice",
+                                "redundancy pay (taxable and non-taxable)",
+                                "accrued holiday pay",
+                                "employees in Northern Ireland",
+                                "benefits employees receive through pay, for example, family working tax credit",
+                                "expenses payments for attending meetings, for example, councillors"
+                            ]
+                        }]
+                    }, {
+                        "question": "Redundancies and temporary employees",
+                        "content": [{
+                            "description": "Whether there were any significant changes to the:",
```

Comment:
In the slides this says state im not sure if missing one word makes a difference